Resources/config/routing_pjax.yml

```diff
@@ -147,7 +147,7 @@ admin_roleList:
         _controller: ezsystems.platformui.controller.role:listRolesAction
 
 admin_roleView:
-    path: /role/view/{roleId}
+    path: /role/view/{roleId}/{tabId}
```

Comment:
@dpobel The section view code you are referring to does not load the correct tab either. I agree with the reuse of code, but we have to come up with functioning code first.